lib/babelish/android2csv.rb

```diff
@@ -1,5 +1,5 @@
 module Babelish
-  require 'xmlsimple'
+  require 'nokogiri'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.